docs/sources/shared/configuration.md

```diff
@@ -2368,6 +2368,31 @@ otlp_config:
 # Enable writes to Ingesters during Push requests. Defaults to true.
 # CLI flag: -distributor.ingester-writes-enabled
 [ingester_writes_enabled: <boolean> | default = true]
+
+tenant_topic:
+  # Enable the tenant topic tee
+  # CLI flag: -distributor.tenant-topic-tee.enabled
+  [enabled: <boolean> | default = false]
+
+  # Prefix to prepend to tenant IDs to form the final Kafka topic name
+  # CLI flag: -distributor.tenant-topic-tee.topic-prefix
+  [topicprefix: <string> | default = "loki.tenant"]
+
+  # Maximum number of bytes that can be buffered before producing to Kafka
+  # CLI flag: -distributor.tenant-topic-tee.max-buffered-bytes
+  [maxbufferedbytes: <int> | default = 100MiB]
+
+  # Maximum amount of time to wait before sending a batch to Kafka
+  # CLI flag: -distributor.tenant-topic-tee.batch-timeout
+  [batchtimeout: <duration> | default = 10s]
+
+  # Maximum size of a single Kafka record in bytes
+  # CLI flag: -distributor.tenant-topic-tee.max-record-size-bytes
+  [maxrecordsizebytes: <int> | default = 15MiB249KiB]
```

Comment:
Is the default a typo? Or is this some developer magic I just haven't encountered before?

Reply:
This is a derived value based on some existing constants in the code, which unfortunately creates a less than pleasant reading experience for the default.

```
	// ProducerBatchMaxBytes is the max allowed size of a batch of Kafka records.
	ProducerBatchMaxBytes = 16_000_000

	// MaxProducerRecordDataBytesLimit is the max allowed size of a single record data. Given we have a limit
	// on the max batch size (ProducerBatchMaxBytes), a Kafka record data can't be bigger than the batch size
	// minus some overhead required to serialise the batch and the record itself. We use 16KB as such overhead
	// in the worst case scenario, which is expected to be way above the actual one.
	MaxProducerRecordDataBytesLimit = ProducerBatchMaxBytes - 16384
```